### PR TITLE
1/first branch

### PR DESCRIPTION
**Pull request by KozhanSerhii**

### CODE REVIEW

File: Infrastructure/Sale.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+public class Sale
+{
+    public long Id { get; set; }
+    public string? Author { get; set; }
+    public string? Book { get; set; }
+    public long Price { get; set; }
+    public long Number_of_sales { get; set; }
+}
+
```
Comment (KozhanMykhailo) [inline Infrastructure/Sale.cs]: empty row should be deleted

File: Infrastructure/BookStoreContext.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace ConsoleBookStore
+{
+    public partial class BookStoreContext : DbContext
+    {
+        public BookStoreContext()
+        {
+        }
+        public BookStoreContext(DbContextOptions<BookStoreContext> options)
+            : base(options)
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:14]: refactor of brackets

File: Infrastructure/BookStoreContext.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        {
+        }
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:17]: add whitespace after bracket

File: Infrastructure/BookStoreContext.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        public virtual DbSet<Sale> Sales { get; set; } = null!;
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                #warning        To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see http://go.microsoft.com/fwlink/?LinkId=723263.
+                optionsBuilder.UseSqlite("Data Source=BookStore.db");
+            }
+        }
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:27]: add whitespace after bracket

File: Infrastructure/BookStoreContext.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            OnModelCreatingPartial(modelBuilder);
+        }
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:32]: add whitespace after bracket

File: Infrastructure/BookStoreContext.cs [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+namespace ConsoleBookStore
+{
+    public partial class BookStoreContext : DbContext
+    {
+        public BookStoreContext()
+        {
+        }
+        public BookStoreContext(DbContextOptions<BookStoreContext> options)
+            : base(options)
+        {
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:15]: refactor signature row

File: Infrastructure/BookStoreContext.cs [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace ConsoleBookStore
+{
+    public partial class BookStoreContext : DbContext
+    {
+        public BookStoreContext()
+        {
+        }
```
Comment (KozhanMykhailo) [inline Infrastructure/BookStoreContext.cs:13]: add whitespace

**Comment by KozhanSerhii** [comment]
done

**Pull request merged**